src/domain/accounts/index.types.d.ts

```diff
@@ -37,9 +37,23 @@ type BusinessMapMarker = {
   mapInfo: BusinessMapInfo
 }
 
+type LimitsChecker = {
+  checkTwoFA({ pendingAmount }: { pendingAmount: Satoshis }): true | LimitsExceededError
```

Comment:
didn't know we can use true as a type, this is a bit weird and maybe shoud be a custom type or just boolean.

Reply:
Recommendation from @samerbuna